assets/src/edit-story/app/story/useStoryReducer/actions.js

```diff
@@ -74,6 +74,12 @@ const updateElementById = (dispatch) => ({ elementId, properties }) =>
     payload: { elementIds: [elementId], properties },
   });
 
+const updateElementsByVideoId = (dispatch) => ({ videoId, properties }) =>
```

Comment:
The story reducers are thoroughly tested. `updateElementsByVideoId` needs test coverage too.

Reply:
Test cases are added!